week2/class2/Program.cs

```diff
@@ -0,0 +1,68 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+//Caclulator
+app.MapGet("/calc", (string numberOne,string numberTwo,string operation) => calcFun(numberOne,numberTwo,operation));
+string calcFun(string numberOne,string numberTwo,string operation)

```

Comment:
compared to `JS`, in `C#` is much more common to use PascalCase when naming methods, e.g `calcFun => CalcFun` 😊

Reply:
Thank you .Done